satellite/audit/service.go

```diff
@@ -72,6 +78,31 @@ func (service *Service) Run(ctx context.Context) (err error) {
 	})
 }
 
+// RemoteSegment takes a remote segment found in metainfo and creates a reservoir for it if it doesn't exist already
+func (service *Service) RemoteSegment(ctx context.Context, path storj.Path, pointer *pb.Pointer) (err error) {
```

Comment:
Do you think it would be better to make the audit service implement the `Observer` interface, or to create a separate struct for this? Like `reservoirObserver` or something.

Reply:
yeah that would probably be cleaner